### PR TITLE
Remove patch

### PR DESCRIPTION
**Pull request by Amoghrd**
Remove patch for eBPF
Sleep 60s
Remove deletion of non-existent operatorGroup.yaml file

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -388,8 +388,7 @@ pipeline {
                     env.RELEASE = sh(returnStdout: true, script: "oc get pods -l app=netobserv-operator -o jsonpath='{.items[*].spec.containers[1].env[0].value}' -n ${NAMESPACE}").trim()
                     returnCode = sh(returnStatus: true, script: """
                         oc -n $NAMESPACE patch csv $RELEASE --type=json -p "[{"op": "replace", "path": "/spec/install/spec/deployments/0/spec/template/spec/containers/0/resources/limits/memory", "value": ${params.CONTROLLER_MEMORY_LIMIT}}]"
-                        oc patch flowcollector cluster --type=json -p "[{"op": "replace", "path": "/spec/agent/type", "value": "EBPF"}] -n netobserv"
-                        oc patch flowcollector cluster --type=json -p "[{"op": "replace", "path": "/spec/agent/ebpf/sampling", "value": ${params.FLP_SAMPLING_RATE}}] -n netobserv"
+			sleep(time:60,unit:"SECONDS")
```
Comment (nathan-weinberg) [inline Jenkinsfile:391]: Since this is a shell block, we can just do a regular `sleep 60`

**Review by nathan-weinberg — CHANGES_REQUESTED** [review]

File: Jenkinsfile [hunk 2]
```diff
@@ -388,8 +388,7 @@ pipeline {
                     env.RELEASE = sh(returnStdout: true, script: "oc get pods -l app=netobserv-operator -o jsonpath='{.items[*].spec.containers[1].env[0].value}' -n ${NAMESPACE}").trim()
                     returnCode = sh(returnStatus: true, script: """
                         oc -n $NAMESPACE patch csv $RELEASE --type=json -p "[{"op": "replace", "path": "/spec/install/spec/deployments/0/spec/template/spec/containers/0/resources/limits/memory", "value": ${params.CONTROLLER_MEMORY_LIMIT}}]"
-                        oc patch flowcollector cluster --type=json -p "[{"op": "replace", "path": "/spec/agent/type", "value": "EBPF"}] -n netobserv"
-                        oc patch flowcollector cluster --type=json -p "[{"op": "replace", "path": "/spec/agent/ebpf/sampling", "value": ${params.FLP_SAMPLING_RATE}}] -n netobserv"
```
Comment (nathan-weinberg) [inline Jenkinsfile:392]: This should not be removed AFAIK

**Review by nathan-weinberg — CHANGES_REQUESTED** [review]

**Comment by nathan-weinberg** [comment]
Changes look fine, please add a success run

**Comment by Amoghrd** [comment]
Thats what I need your help on in the meeting😅

**Comment by Amoghrd** [comment]
Success run: [10](https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/aramesha-e2e-benchmarking-multibranch-pipeline/job/update-patch/10/console)

**Review by nathan-weinberg — APPROVED** [review]
/lgtm

**Pull request merged**